docs/content/configuring-npm/package-json.md

```diff
@@ -718,6 +718,30 @@ the host package's major version will break your plugin. Thus, if you've worked
 with every 1.x version of the host package, use `"^1.0"` or `"1.x"` to express
 this. If you depend on features introduced in 1.5.2, use `">= 1.5.2 < 2"`.
 
+### peerDependenciesMeta
+
+When a user installs a your package, npm will emit warnings if packages specified in `peerDependencies` are not already installed. The `peerDependenciesMeta` field serves to provide npm more information on how your peer dependencies are to be used. Specifically, it allows peer dependencies to be marked as optional.
```

Comment:
Tiny typo, change "When a user installs **a** your package" to "When a user installs your package".

Reply:
Good find, thank you for reviewing! I've fixed it 😄